include/riak_cs_gc.hrl

```diff
@@ -106,3 +93,14 @@
 -define(DEFAULT_GC_BATCH_SIZE, 1000).
 -define(DEFAULT_GC_WORKERS, 2).
 -define(EPOCH_START, <<"0">>).
+
+-record(gc_manager_state, {
+          next :: undefined | non_neg_integer(),
+          gc_batch_pid :: undefined | pid(),
+          batch_history = [] :: list(#gc_batch_state{}),
```

Comment:
Wouldn't it be good if we can know stats of three or ten last run of garbage collection?